README.md

```diff
@@ -184,4 +185,13 @@ Console.WriteLine("AB.3(16) = " + y + "(1)");
 >>> AB.3(16) = 171,1875(1)
 ```
 
-See more on [Wiki](https://github.com/asc-community/AngouriMath/wiki).
\ No newline at end of file
+See more on [Wiki](https://github.com/asc-community/AngouriMath/wiki).
+
+
+### <a name="contrib"></a>Contribution
+
+We appreciate and welcome any contributors to AngouriMath.
+
+Use pull requests to contribute to it. If you want to regenerate the parser, follow these steps:
```

Comment:
To be more clear, specify what files constitute the parser.

Reply:
ok